client/src/components/Overview/subcomponents/Category.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const Category = ({ product }) => (
+  <div>
+    <div>{product.category}</div>
```

Comment:
I would suggest to use text html tags like `<p>` or `headers(<h1>)` instead using `<div>`